src/env-variables.ts

```diff
@@ -11,7 +11,7 @@ const WORKOS_COOKIE_DOMAIN = getEnvVariable('WORKOS_COOKIE_DOMAIN');
 const WORKOS_COOKIE_MAX_AGE = getEnvVariable('WORKOS_COOKIE_MAX_AGE');
 const WORKOS_COOKIE_NAME = getEnvVariable('WORKOS_COOKIE_NAME');
 const WORKOS_COOKIE_PASSWORD = getEnvVariable('WORKOS_COOKIE_PASSWORD') ?? '';
-const WORKOS_REDIRECT_URI = getEnvVariable('NEXT_PUBLIC_WORKOS_REDIRECT_URI') ?? '';
+const WORKOS_REDIRECT_URI = process.env.NEXT_PUBLIC_WORKOS_REDIRECT_URI ?? '';
```

Comment:
This is baffling, I wonder why `process.env[name]` isn't working as expected. Since `getEnvVariable` doesn't add much anymore, would you mind replacing every instance in this file with a direct lookup of `process.env`? e.g. 
```suggestion
const WORKOS_API_HOSTNAME = process.env..WORKOS_API_HOSTNAME;
const WORKOS_API_HTTPS = process.env.WORKOS_API_HTTPS;
const WORKOS_API_KEY = process.env.WORKOS_API_KEY ?? '';
const WORKOS_API_PORT = process.env.WORKOS_API_PORT;
const WORKOS_CLIENT_ID = process.env.WORKOS_CLIENT_ID ?? '';
const WORKOS_COOKIE_DOMAIN = process.env.WORKOS_COOKIE_DOMAIN;
const WORKOS_COOKIE_MAX_AGE = process.env.WORKOS_COOKIE_MAX_AGE;
const WORKOS_COOKIE_NAME = process.env.WORKOS_COOKIE_NAME;
const WORKOS_COOKIE_PASSWORD = process.env.WORKOS_COOKIE_PASSWORD ?? '';
const WORKOS_REDIRECT_URI = process.env.NEXT_PUBLIC_WORKOS_REDIRECT_URI ?? '';
```

Reply:
Didn't see the nextjs docs you linked, makes sense now. Thanks for catching!